WebDriverAgentLib/Routing/FBElementUtils.m

```diff
@@ -111,15 +111,23 @@ + (NSString *)wdAttributeNameForAttributeName:(NSString *)name
 
 static BOOL FBShouldUsePayloadForUIDExtraction = YES;
 static dispatch_once_t oncePayloadToken;
-+ (NSUInteger)uidWithAccessibilityElement:(XCAccessibilityElement *)element
++ (NSString *)uidWithAccessibilityElement:(XCAccessibilityElement *)element
 {
   dispatch_once(&oncePayloadToken, ^{
     FBShouldUsePayloadForUIDExtraction = [element respondsToSelector:@selector(payload)];
   });
+  unsigned long long elementId;
   if (FBShouldUsePayloadForUIDExtraction) {
-    return [[element.payload objectForKey:@"uid.elementID"] intValue];
+    elementId = [[element.payload objectForKey:@"uid.elementID"] longLongValue];
+  } else {
+    elementId = [[element valueForKey:@"_elementID"] longLongValue];
   }
-  return [[element valueForKey:@"_elementID"] intValue];
+  int processId = element.processIdentifier;
+  uint8_t b[16] = {0};
+  memcpy(b, &elementId, sizeof(long long));
+  memcpy(b + sizeof(long long), &processId, sizeof(int));
```

Comment:
I was thinking about this yesterday; perhaps you want to assert that `sizeof(long long) + sizeof(int) < 16`, although that's guaranteed to be the case on 32-bit and 64-bit platforms.